nuget.config

```diff
@@ -1,7 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <packageSources>
-        <add key="NuGet official package source" value="https://api.nuget.org/v3/index.json" />
-        <add key="Local Packages" value="src/Files.App/nupkgs/" />
+        <add key="Project Packages" value="src/Files.App/Assets/Libraries/" />
```

Comment:
```suggestion
        <add key="Local Packages" value="src/Files.App/Assets/Libraries/" />
```

Reply:
"Local Packages" is also the name of the Office offline packages store.

Reply:
Do we use that?

Reply:
I see. Btw we should distribute through NuGet.org tho, this way is way much easier to update package.